client/src/pages/Login/Login.css

```diff
@@ -1,8 +1,185 @@
-.login-container{
+*{
+    margin: 0;
+    padding: 0;
+  }
+
+  .Login-container{
     display: flex;
     height: 80vh;
-    justify-content: center;
-    align-items: center;
-    font-family: "RobotoSlab-Variable";
+    font-family: RobotoSlab-VariableFont_wght;
     text-transform: uppercase;
+    padding: 0;
+    margin: 0;
+  }
+  .Login-form-div{
+  flex: 830px;
+  flex-direction: column;
+  background-color:#FFF8F3 ;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+    height: 880px;
+  }
+  
+  .Login-img-div{
+  height: 899px;
+  flex: 510px;
+  padding: 0;
+  margin: 0;  
+  }
+  .Login_img{
+  height: 899px;
+  width: 510px;
+  margin: 0;
+  padding: 0;
+  background-position: cover;
+  background-size: cover;
+  background-repeat: no-repeat;
+  }
+  
+  
+  .Login-form{  
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    width: 450px;
+    height: 310px;
+    
+  } 
+  .p-Login{
+  display: flex;
+  size: 48px;
+  width: 450px;
+  height: 63px;
+  font: var(--unnamed-font-style-normal) normal var(--unnamed-font-weight-normal) var(--unnamed-font-size-48)/var(--unnamed-line-spacing-64) var(--unnamed-font-family-roboto-slab);
+  letter-spacing: var(--unnamed-character-spacing-0-48);
+  color: var(--unnamed-color-0f2420);
+  text-align: left;
+  font: normal normal normal 42px/64px Roboto Slab;
+  letter-spacing: 0.45px;
+  color: #0F2420;
+  opacity: 1;
+  
+  }
+  
+  .input-form{
+    margin: 5px  ;
+    height: 46px;
+    border: 1px solid #D9DEEA;
+    border-radius: 4px;
+    opacity: 1;
+    width: 380px;
+  }
+  
+  
+  ::placeholder {
```

Comment:
you need to target particular element so use the correct class name otherwise it will do this style for all placeholders

Reply:
i did not find any way to specify a certain or particular styling for placeholder!
anyway, our place are having the sma style